src/ExchangeApi.ts

```diff
@@ -133,27 +136,38 @@ export default class ExchangeApi {
         this.loginEndPoint = loginEndPoint;
     }
 
-    public async login(username: string, password: string): Promise<boolean> {
-        this.authToken = '';
-        let success = false;
-
-        const resp = await fetch(this.loginEndPoint, {
-            body: `username=${username}&password=${password}`,
-            headers: {
-                'Accept': 'application/json',
-                'Accept-Encoding': 'gzip',
-                'Content-Type': 'application/x-www-form-urlencoded',
-                'X-Application': this.applicationKey,
-            },
-            method: 'POST',
+    public async login(
+        username: string,
+        password: string,
+        betfairPrivateKey?: string,
+        betfairPublicCert?: string,
```

Comment:
Discussion point here / breaking change. In theory I've made these mandatory for this endpoint.

What's the non-argument way of getting this working?

This way is:

- Create keys + certs
- Base64 them to single-line env-vars
- Import via env-var and decode to original values
- Pass certs to login function